spec/controllers/api/v1/media_controller_spec.rb

```diff
@@ -10,7 +10,7 @@
     allow(controller).to receive(:doorkeeper_token) { token }
   end
 
-  describe 'POST #create' do
+  xdescribe 'POST #create' do
```

Comment:
@d3cline 

Took me some moment to fiddle a working mock setup for this spec.

```suggestion
  describe 'POST #create' do
    describe 'with paperclip errors' do
      let(:media_attachment) { double(MediaAttachment) }
      let(:account) { Fabricate(:account) }
      let(:token) { Fabricate(:accessible_access_token, resource_owner_id: account.user.id, scopes: 'write:media') }
    
      before do
        allow(controller).to receive(:doorkeeper_token) { token }
        allow(MediaAttachment).to receive(:new).and_return(media_attachment)
        %i[_has_attribute? new_record? _write_attribute create! association].each do |method|
          allow(media_attachment).to receive(method)
        end
      end
      context 'when imagemagick cant identify the file type' do
        it 'returns http 422' do
          expect(media_attachment).to receive(:save!).and_raise(Paperclip::Errors::NotIdentifiedByImageMagickError)
          post :create, params: { file: fixture_file_upload('attachment.jpg', 'image/jpeg') }
          expect(response).to have_http_status(:unprocessable_entity)
        end
      end

      context 'when there is a generic error' do
        it 'returns http 422' do
          expect(media_attachment).to receive(:save!).and_raise(Paperclip::Error)
          post :create, params: { file: fixture_file_upload('attachment.jpg', 'image/jpeg') }
          expect(response).to have_http_status(500)
        end
      end
    end
```

The problem was, that this stub was not done in a manner that it should have been to begin with.

The use of expect_any_instance_of already pointed into that direction.

The [Ruby style guide disregards its use for a reason](https://rspec.rubystyle.guide/#any_instance_of) :shrug:

I hope that helped you.

For the future:
Skipping specs must have a good reason, which usually should be that they're obsolete or replaced by another spec.

Reply:
Thank you so much for the help, I was lost but felt like I was on the right track. I hit a point where to figure it out would have been a large investment of time to understand the scope. Thanks :D